src/services/e2e.ts

```diff
@@ -41,6 +44,15 @@ export const services = <T extends Record<string, Token>>(app: T): ServiceDefini
   [app.cy, {
     constant: cy,
   }],
+  [$.client, {
+    service: () => {
+      return {
+        history: [],
+      }
+    },
+    arguments: [
+    ],
+  }],
```

Comment:
Again this is likely to be moved to a more formal 'client' once the "is request counting advantageous or not" is clearer.